<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Main_Table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>10</pointsize>
      <weight>12</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 255, 127);
color: rgb(0, 85, 5);
font: 100 10pt &quot;Palatino Linotype&quot;;
background-color: rgb(255, 235, 255);
background-color: rgb(209, 209, 209);
</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>5</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="Page1">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>481</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Asap,Helvetica Neue,Helvetica,sans-serif'; font-size:12pt; font-weight:600; color:#000000;&quot;&gt;VISUALISATION ET ANALISE DES DONNEES D'ENERGIE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>381</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Asap,Helvetica Neue,Helvetica,sans-serif'; font-size:xx-large; color:#005500; text-transform:uppercase;&quot;&gt;Le réseau au coeur &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Asap,Helvetica Neue,Helvetica,sans-serif'; font-size:xx-large; color:#005500; text-transform:uppercase;&quot;&gt;des données d'énergie&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-family:'Asap,Helvetica Neue,Helvetica,sans-serif'; font-weight:600; color:#005500;&quot;&gt;IUT De Roanne:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-family:'Asap,Helvetica Neue,Helvetica,sans-serif'; font-weight:600; color:#005500;&quot;&gt;By SYFA05&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Asap,Helvetica Neue,Helvetica,sans-serif'; font-size:14pt; font-weight:600; color:#005500;&quot;&gt;Consommation quotidienne brute&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>440</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Asap,Helvetica Neue,Helvetica,sans-serif'; font-weight:600; color:#005500;&quot;&gt;ODRE:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Asap,Helvetica Neue,Helvetica,sans-serif'; font-weight:600; color:#005500;&quot;&gt;OpenData Reseaux-Energies&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Asap,Helvetica Neue,Helvetica,sans-serif'; font-size:10pt; font-weight:400; color:#000000;&quot;&gt;Explorez, comprenez, et appropriez vous les données d’énergie,&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Asap,Helvetica Neue,Helvetica,sans-serif'; font-size:10pt; font-weight:400; color:#000000;&quot;&gt;En direct des transporteurs d’énergie et de leurs partenaires&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_1">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>350</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>51</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CONTINUER</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Page2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Palatino Linotype'; font-size:10pt; font-weight:96; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Importation des fichiers&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Importer fichier .xls</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>77</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>RUN</string>
      </property>
      <property name="iconSize">
       <size>
        <width>11</width>
        <height>16</height>
       </size>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Importer  via URL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Debut</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>460</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Precedent</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>311</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>460</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>CommandLinkButton</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="currentText">
       <string>---Interpolation---</string>
      </property>
      <item>
       <property name="text">
        <string>---Interpolation---</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Consommation brute gaz (MW PCS 0°C) - GRTgaz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Consommation brute gaz (MW PCS 0°C) - Teréga</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Consommation brute gaz totale (MW PCS 0°C)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Consommation brute électricité (MW) - RTE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Consommation brute totale (MW)</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>DATA Set</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>330</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>380</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Valeur MAX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Valeur MIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Valeue Moyenne</string>
      </property>
     </widget>
     <widget class="QOpenGLWidget" name="openGLWidget">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>150</y>
        <width>501</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>120</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Histogramme des valeurs moyennes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Page3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Palatino Linotype'; font-size:10pt; font-weight:96; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Exploitation des donneés&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>DATA Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Precedent</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="currentText">
       <string>---Interpolation---</string>
      </property>
      <item>
       <property name="text">
        <string>---Interpolation---</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Consommation brute gaz (MW PCS 0°C) - GRTgaz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Consommation brute gaz (MW PCS 0°C) - Teréga</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Consommation brute gaz totale (MW PCS 0°C)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Consommation brute électricité (MW) - RTE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Consommation brute totale (MW)</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>77</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>RUN</string>
      </property>
      <property name="iconSize">
       <size>
        <width>11</width>
        <height>16</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>460</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>10</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
